.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,62 @@
+name: "Bug Report"
+description: Report a bug to help improve blueking-apigateway.
+title: "bug: ......"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        _提供足够详细的信息, 以便我们能够快速帮助你__
+
+        _The more information you share, the faster we can help you._
+
+        提交 Issue 前, 请先搜索 [开放的 Issue](https://github.com/TencentBlueKing/blueking-apigateway/issues), 避免重复提交.
+
+        Search the [open issues](https://github.com/apache/apisix/issues) first before you open a new issue.
+
+  - type: textarea
+    id: current-behavior
+    attributes:
+      label: Current Behavior
+      description: 遇到的问题(Describe the issue you are facing)
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected Behavior
+      description: 预期行为(Describe what you expected to happen)
+    validations:
+      required: false
+  - type: textarea
+    id: error
+    attributes:
+      label: Error Logs
+      description: 错误日志(Paste the error logs if any)
+    validations:
+      required: false
+  - type: textarea
+    id: steps
+    attributes:
+      label: Steps to Reproduce
+      description: 复现步骤(Share the steps you took so that we can reproduce the issue. Reports without proper steps details will likely be closed.)
+      placeholder: |
+        1. ...
+        2. ...
+        3. ...
+    validations:
+      required: true
+  - type: textarea
+    id: version-environment
+    attributes:
+      label: Version and Environment
+      description: 版本及环境信息(Share your version and environment details)
+      value: |
+        - 蓝鲸版本号(blueking version):
+        - 容器化部署还是二进制部署(containerized or binary):
+        - 二进制部署请提供网关版本号(binary version of bk-apigateway):
```

Comment:
binary version of bk-apigateway -> The version of bk-apigateway, required for binary distribution